k8s/perftest/common/divorce/div-cms.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: div-cms
+  namespace: divorce
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: div-cms
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: div-cms
+    version: 1.1.0
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
+      image: hmctspublic.azurecr.io/div/cms:prod-f7412e3e
+      ingressHost: div-cms-perftest.service.core-compute-perftest.internal
+      environment:
+        AUTH_PROVIDER_SERVICE_CLIENT_BASEURL : "http://rpe-service-auth-provider-perftest.service.core-compute-perftest.internal"
+        DRAFT_STORE_API_BASEURL : "http://draft-store-service-perftest.service.core-compute-perftest.internal"
+        IDAM_CASEWORKER_USERNAME : "DivCaseWorker@AAT.com"
+        CASE_FORMATTER_SERVICE_API_BASEURL : "http://div-cfs-perftest.service.core-compute-perftest.internal"
+        IDAM_API_BASEURL: "https://idam-api.aat.platform.hmcts.net"
```

Comment:
should this be perftest too?

Reply:
Yep good spot thanks

Reply:
@qzhou-hmcts  https://github.com/hmcts/cnp-flux-config/pull/482